composer.json

```diff
@@ -0,0 +1,62 @@
+
+{
+    "name": "phpredis/phpredis",
```

Comment:
Please verify vendor namespace & package name are acceptable

Reply:
Seems right to me, although I need to read up on PIE